x-pack/plugins/apm/public/components/app/Settings/schema/schema.stories.tsx

```diff
@@ -15,21 +16,80 @@ import { Schema } from './';
 export default {
   title: 'app/Settings/Schema',
   component: Schema,
+  argTypes: {
+    hasCloudAgentPolicy: {
+      control: {
+        type: 'boolean',
+        options: [true, false],
+        defaultValue: true,
+      },
+    },
+    hasCloudApmPackagePolicy: {
+      control: {
+        type: 'boolean',
+        options: [true, false],
+        defaultValue: false,
+      },
+    },
+    cloudApmMigrationEnabled: {
+      control: {
+        type: 'boolean',
+        options: [true, false],
+        defaultValue: true,
+      },
+    },
+    hasRequiredRole: {
+      control: {
+        type: 'boolean',
+        options: [true, false],
+        defaultValue: true,
+      },
+    },
+    isMigrating: {
+      control: {
+        type: 'boolean',
+        options: [true, false],
+        defaultValue: false,
+      },
+    },
+  },
   decorators: [
-    (StoryComponent: ComponentType) => {
+    (StoryComponent: ComponentType, { args }: any) => {
```

Comment:
```suggestion
    (StoryComponent: ComponentType, { args }: Meta<Args>) => {
```

The `Meta` type can be imported alongside `Story`, and `Args` can be an interface you define the matches the arg types.